src/components/com_tjcertificate/administrator/controllers/certificates.php

```diff
@@ -34,4 +38,107 @@ public function getModel($name = 'Certificate', $prefix = 'TjCertificateModel')
 	{
 		return parent::getModel($name, $prefix, array('ignore_request' => true));
 	}
+
+	/**
+	 * Method to delete the record from frontend.
+	 *
+	 * @return  void|boolean
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function deleteCertificate()
+	{
+		$user = Factory::getUser();
+		$app  = Factory::getApplication();
+		$certificateId = $app->input->getInt('id');
+		$manageOwn = $user->authorise('certificate.external.manageown', 'com_tjcertificate');
+		$manage    = $user->authorise('certificate.external.manage', 'com_tjcertificate');
+
+		// If manageOwn permission then check record owner can only deleting own record
+		if ($manageOwn && !$manage)
```

Comment:
Change to `if ($manageOwn)`

Reply:
by default admin have all the permission so for admin also if condition will check